UnityProject/Assets/Scripts/Items/Bureaucracy/PaperBin.cs

```diff
@@ -0,0 +1,250 @@
+using System;
+using Mirror;
+using UnityEngine;
+
+namespace Items.Bureaucracy
+{
+	public class PaperBin : NBHandApplyInteractable, IOnStageServer
+	{
+
+		public int initialPaperCount = 20;
+		public GameObject blankPaper;
+
+		[SyncVar (hook = nameof(SyncBlankPaperCount))]
+		private int blankPaperCount;
+
+		private readonly SyncListPaper storedPaper = new SyncListPaper();
```

Comment:
Why should all clients know contents of each paper bin on the server? (syncvars are public for all clients)
Look at how closet keeps its contents secret from player when it's closed. 
Syncvars are okay for visual appearance, but not for hidden stuff

Reply:
No longer being synced